stack-8.2.2.yaml

```diff
@@ -1,6 +1,9 @@
 resolver: nightly-2017-11-24
```

Comment:
@vrom911 I'm not sure why build failed. But probably this is because `stack-8.2.2.yaml` uses old `nigthly` snapshot. Could you try to change it to `importify` version?

https://github.com/serokell/importify/blob/e29429faed84ea104f064ac2d4fbd27aa5993ea2/stack-8.2.2.yaml#L1

I'm not sure this will fix... Because it worked earlier with `stack-8.2.2.yaml`.